frontend/lib/category_assets.dart

```diff
@@ -30,157 +29,159 @@ class CategoryAsset {
+      shortName: context.l10n.category_media,
+      icon: 'assets/category_icons/1.svg',
+      detailIcon: 'assets/detail_headers/1_multimedia.svg',
+      color: Color(0xFFFA0000),
+    ),
+    CategoryAsset(
+      id: 2,
+      name: context.l10n.category_healthLong,
+      shortName: context.l10n.category_health,
+      icon: 'assets/category_icons/2.svg',
+      detailIcon: 'assets/detail_headers/2_sport.svg',
+      color: Color(0xFFE500D3),
+    ),
+    CategoryAsset(
+      id: 3,
+      name: context.l10n.category_cultureLong,
+      shortName: context.l10n.category_culture,
+      icon: 'assets/category_icons/3.svg',
+      detailIcon: 'assets/detail_headers/3_kultur.svg',
+      color: Color(0xFF7500EB),
+    ),
+    CategoryAsset(
+      id: 4,
+      name: context.l10n.category_servicesLong,
+      shortName: context.l10n.category_services,
+      icon: 'assets/category_icons/4.svg',
+      detailIcon: 'assets/detail_headers/4_finanzen.svg',
+      color: Color(0xFF515151),
+    ),
+    CategoryAsset(
+      id: 5,
+      name: context.l10n.category_fashionLong,
+      shortName: context.l10n.category_fashion,
+      icon: 'assets/category_icons/5.svg',
+      detailIcon: 'assets/detail_headers/5_mode.svg',
+      color: Color(0xFF6EBE00),
+    ),
+    CategoryAsset(
+      id: 6,
+      name: context.l10n.category_livingLong,
+      shortName: context.l10n.category_living,
+      icon: 'assets/category_icons/6.svg',
+      detailIcon: 'assets/detail_headers/6_haus.svg',
+      color: Color(0xFF00D0C7),
+    ),
+    CategoryAsset(
+      id: 7,
+      name: context.l10n.category_leisureLong,
+      shortName: context.l10n.category_leisure,
+      icon: 'assets/category_icons/7.svg',
+      detailIcon: 'assets/detail_headers/7_freizeit.svg',
+      color: Color(0xFF007CE8),
+    ),
+    CategoryAsset(
+      id: 8,
+      name: context.l10n.category_foodLong,
+      shortName: context.l10n.category_food,
+      icon: 'assets/category_icons/8.svg',
+      detailIcon: 'assets/detail_headers/8_essen.svg',
+      color: Color(0xFF197489),
+    ),
+    CategoryAsset(
+      id: 9,
+      name: context.l10n.category_other,
+      shortName: context.l10n.category_other,
+      icon: 'assets/category_icons/9.svg',
+      detailIcon: null,
+      color: Color(0xFFc51162),
+    ),
+    CategoryAsset(
+      id: 10,
+      name: context.l10n.category_lunchTables,
+      shortName: context.l10n.category_lunchTables,
+      icon: 'assets/category_icons/10.svg',
+      detailIcon: 'assets/detail_headers/10_mittagstische.svg',
+      color: Color(0xFF197489),
+    ),
+    CategoryAsset(
+      id: 11,
+      name: context.l10n.category_clothingLong,
+      shortName: context.l10n.category_clothing,
+      icon: 'assets/category_icons/11.svg',
+      detailIcon: 'assets/detail_headers/11_kleidung.svg',
+      color: Color(0xFF6EBE00),
+    ),
+    CategoryAsset(
+      id: 12,
+      name: context.l10n.category_cultureLongAlternative,
```

Comment:
Because the categories 3 and 12 are both culture:
``` dart
  CategoryAsset(
    id: 3,
    name: 'Bildung/Kultur/Unterhaltung',
    shortName: 'Kultur',
  ), 
  CategoryAsset(
    name: 'Kultur/Museen/Freizeit',
    shortName: 'Kultur',
```

If you have other ideas let me know.

Reply:
The one with id 12 is a Nürnberg category. So we could call it `cultureLongNuernberg` for better understanding.